core/domain/exp_services.py

```diff
@@ -1270,3 +1273,44 @@ def search_explorations(query, limit, sort=None, cursor=None):
     """
     return search_services.search(
         query, SEARCH_INDEX_EXPLORATIONS, cursor, limit, sort, ids_only=True)
+
+         
+def _is_suggestion_valid(thread_id, exploration_id): 
+    """Check if the suggestion is still valid. A suggestion is considered 
+    invalid if the exploration version has updated since the suggestion was 
+    made.""" 
+ 
+    states = exp_models.ExplorationModel.get(id=exploration_id).states 
```

Comment:
Also, do get_exploration_by_id(exploration_id) instead. (There's a function in this file for that.)